### DISCUSSION TITLE
Eye of the Beholder AGA versions

### DISCUSSION THREAD

**Discussion by codeflorist** [vAmigaWeb/vAmigaWeb#222] [Q&A]
hey there!

i'd love to play the AGA versions of EOB1 and 2 on my Windows surface tablet. is this even possible or viable with vAmigaWeb?

i've downloaded the games in various variants - coming as lha files. when i load the file in vAmigaWeb, i get an error, that the file is too large to be mounted as a disk.

if anyone can help me get this to work, i'd very much appreciate it. many thanks in advance!

**Comment by mithrendal** [comment]
Hey!

Lha files - support is somewhat limited… when throwing a .lha file (or any other unknown media) into vAmigaWeb it will try to import that into a floppy disk… the maximum capacity of a HD floppy disk is 1760 Kb ( = 1.71MB ). If the lha file is smaller than 1,71MB it will be mounted on a floppy disk and can be decrunched in AmigaDOS. 

see here for reference
https://www.youtube.com/watch?v=cDXI1ioQ8zU

bigger lha files can not be imported directly. 

one solution to this is to import them into winuae on hd and export for use in vAmigaWeb
+ extract and save it to the hd and export that .hdf file   
+ mount that hdf file into vAmigaWeb

As for AGA. The AGA chipset is not supported by vAmiga. Classic OCS / ECS variants are supported. Sorry. 

One solution to this would be to play the OCS / ECS versions of Eye of the Beholder.


How is vAmigaWeb running on the Surface ? Does it run on an ARM processor ? Is touch related input working well ?

**Reply by codeflorist** [reply]
Hey, thanks for your answer and detailed explanation! The Eye of the Beholder AGA versions are special fan-modified versions, that contain the VGA graphics of the PC versions and most interestingly an automap. It's generally considered the best version of the games. So i'm looking for a way to play that specific version.

So if vAmigaWeb does not support AGA, it's unfortunately not a viable solution for me.

My Surface has an Intel CPU. I did not do much with vAmigaWeb yet, so i can't say much. But i'll let you know, if i have something to report.

**Comment by mithrendal** [comment]
vAmigaWeb has AGA support now. Just select the Amiga 1200 model, or choose the AGA chipset in the settings.
